picasso/src/main/java/com/squareup/picasso3/Picasso.kt

```diff
@@ -378,6 +378,7 @@ class Picasso internal constructor(
     for (deferredRequestCreator in targetToDeferredRequestCreator.values) {
       deferredRequestCreator.cancel()
     }
+    targetToAction.clear()
```

Comment:
note to self: according to Aidan, this is only happening as of v3.  so is this a kotlin conversion bug or just something we never caught in v2?